documentation/modules/creating-storage-mapping.adoc

```diff
@@ -6,24 +6,29 @@
 [id="creating-storage-mapping_{context}"]
 = Creating a storage mapping
 
-You can create a storage mapping by using the {ocp} web console to map source data stores to {virt} storage classes.
+You can create a storage mapping by using the {ocp} web console to map source storage domains to {virt} storage classes.
 
 .Prerequisites
 
-* Source and target providers added to the web console.
+* Source and target providers added to the {ocp} web console.
 * Local and shared persistent storage that support VM migration.
 
 .Procedure
 
-. Click *Mappings*.
-. Click the *Storage* tab and then click *Create mapping*.
-. Enter the *Name* of the storage mapping.
-. Select a *Source provider* and a *Target provider*.
-. If your source provider is VMware, select a *Source datastore* and a *Target storage class*.
+. In the {ocp} web console, click *Migration* -> *StorageMaps for virtualization*.
+. Click *Create StorageMap*.
+. Complete the following fields:
 
+* *Name*: Enter a name to display in the storage mappings list.
+* *Source provider*: Select a source provider.
+* *Target provider*: Select a target provider.
++
+// Are the following two steps still relevant?
+. If your source provider is VMware, select a *Source datastore* and a *Target storage class*.
 . If your source provider is {rhv-full}, select a *Source storage domain* and a *Target storage class*.
 
-. Optional: Click *Add* to create additional storage mappings or to map multiple source data stores or storage domains to a single storage class.
+. Select a one of the *Source storage domains* and one of the *Target storage classes*.
```

Comment:
I think there's some confusion about this part - this sentence is not needed since the user already selected datastore(s) or storage domain(s) above. also "Select **a one** of the..." needs to change if this remains

Reply:
@ahadas I see your points. Will fix.